### ISSUE TITLE
legend label goes off plot

### ISSUE THREAD

**Issue by gdevenyi** [Mouse-Imaging-Centre/MRIcrotome#8]
```
sliceSeries(nrow = 5, ncol=5, dimension = 1, begin=42, end=263) %>% 
anatomy(averageanatomy, low=300, high=1400) %>%
overlay(mincArray(model, "tvalue-site2km"), low = 2.179, high=3.055, symmetric = T, alpha=0.5) %>% 
addtitle("Site 2 km vs Site 80 km") %>%
legend("t-statistics") %>% 
contourSliceIndicator(averageanatomy, c(700, 1400)) %>% 
draw()
```
![image](https://user-images.githubusercontent.com/3001850/51849177-2a131d00-22ed-11e9-94f4-9b13df894602.png)

**Comment by jasonlerch** [comment]
Yes - known problem if there are too many digits in legend labels. So that’s the easy fix (i.e. 2.2 will do, 2.179 is overkill); not quite sure how to fix that properly, as very high digit numbers on labels make for very awkward looking figures.

________________________________

This e-mail may contain confidential, personal and/or health information(information which may be subject to legal restrictions on use, retention and/or disclosure) for the sole use of the intended recipient. Any review or distribution by anyone other than the person for whom it was originally intended is strictly prohibited. If you have received this e-mail in error, please contact the sender and delete all copies.

**Comment by gdevenyi** [comment]
Thanks, adjusting the overlay low and high does indeed fix the misalignment.

**Comment by gdevenyi** [comment]
Following up here, there's still a bit of an issue with the vertical spacing, the bottom label is immediately adjacent to the bottom of the page, making it "feel" cutoff. A little bit of padding there would fix that.

**Comment by jasonlerch** [comment]
Makes sense, and we’ll add that. Can be worked around with your own viewports at the moment if immediately necessary.

________________________________

This e-mail may contain confidential, personal and/or health information(information which may be subject to legal restrictions on use, retention and/or disclosure) for the sole use of the intended recipient. Any review or distribution by anyone other than the person for whom it was originally intended is strictly prohibited. If you have received this e-mail in error, please contact the sender and delete all copies.

**Comment by gdevenyi** [comment]
Having some odd issues with this again, weirdly inconsistent?
![human_residuals](https://user-images.githubusercontent.com/3001850/92163969-8a8c0980-ee02-11ea-8a6b-7f294984f2bb.png)
![chimp_residuals](https://user-images.githubusercontent.com/3001850/92163972-8b24a000-ee02-11ea-8079-a99613c0fe71.png)

**Comment by gdevenyi** [comment]
A followup here, exporting these in SVG, the text actually exists and runs off the page (defined by the white square background). One can then resize to fix the cutoff components after the fact during export to PNG from inkscape or perhaps other tools by telling the  tool to consider the dimensions of the drawing rather than just the page size.
